src/main/java/sirius/biz/web/BasePageHelper.java

```diff
@@ -392,6 +388,19 @@ public void iterateAllTotalItems(Consumer<E> consumer) {
         });
     }
 
+    /**
+     * Returns the underlying query for this page helper including all given facets.
+     *
+     * @return the {@link Query} object
+     */
+    public Q buildUnderlyingQuery() {
```

Comment:
couldn't we then kill iterate and iterateAll as these as 100% redundant to query.iteratAll etc...

Reply:
Sound logical. Will rip'em off...